DEVELOPMENT.md

```diff
@@ -14,6 +47,45 @@ Repository contains repository hook definition to auto-generate chart documentat
 When you have the pre-commit hook installed and `commit` a file, it will run the pre-commit hook. If there is a change generated in the `README.md`, the hook will fail and notify you about the new change. You can review the changes and run the `commit` again, now it should pass.
 
 
+## Build and test the project
```

Comment:
Im really not sure if we should be advertising this stuff. `helm_install.sh` is something **we use only for CI**, and isnt used for "development" of the charts per se. It's caused problems/confusion in the past and Ive stopped using it myself. To be honest if a turn key solution is what we're trying to provide here then we should wait for the official Terraform scripts.